index.html

```diff
@@ -0,0 +1,50 @@
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" type="text/css" href="styles.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.1/css/all.min.css" integrity="sha512-u2Qo5c5wL2fZ5m0z/rqZjLmZ9+vHjE/AqY0rU/QdugnfeE0k8zO6rU2qzUITjtWc6i1Kc2nh/gPmKv5fK/xJwg==" crossorigin="anonymous" >
+
+  </head>
+  <body>
+    <div class="portfolio">
+        <header class="details">
+            <div>
+            <img src="images/Time Style.png" alt="time" class="top-left-image">
+            <img src ="images/Group 4.png" alt="battery" class="top-right-image">
+        </div>
```

Comment:
- [ OPTIONAL ] From the project design, the `time`, `battery`, and `wifi` were not meant to be added to the project but to just show you it is for mobile design. So please kindly remove them from your project. :pray: